### PR TITLE
feat(semantic-releases): use semantic releases

### PR DESCRIPTION
**Pull request by open-dynaMIX**
Additionally add commit message linting to travis.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,10 @@ target/
 
 # IDE specific configurations
 .idea/
+
+# npm
+node_modules/
+package-lock.json
+
+# commitlint
+commitlint.config.js
```
Comment (czosel) [inline .gitignore]: Why is this ignored?

Comment (open-dynaMIX) [reply .gitignore]: You're right. I added this to the .gitignore before realizing we will need this file in the pipeline as well.

Comment (open-dynaMIX) [reply .gitignore]: Fixed

**Pull request merged**